File: beginner/javascript/1564.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.on('line', (input) => {
  const n = parseInt(input);
  if (n === 0)
    console.log("vai ter copa!");
  else
    console.log("vai ter duas!");
});

File: beginner/java/1564.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        short n;

        while (scanner.hasNextShort()) {
            n = scanner.nextShort();
            if (n == 0)
                System.out.println("vai ter copa!");
            else
                System.out.println("vai ter duas!");
        }

        scanner.close();
    }
}
